src/jsLib.gs

```diff
@@ -47,7 +47,7 @@ function reverse(string) {
 function buildUrl(url, parameters) {
   var qs = "";
   for (var key in parameters) {
-    if (!parameters.hasOwnProperty(key)) continue;
+	if (!parameters.hasOwnProperty(key)) continue;
```

Comment:
Typo with the tabs?

Reply:
Dont get me started :)  My office IDE cant apply exact same JS templates as my private one :(
Its supposed to be 2 spaces = 1 tab

Reply:
I've been using Visual Studio code (if you havent) - its free available on all platforms and darn good.